Makefile

```diff
@@ -3,6 +3,7 @@ VERSION = edge
 GIT_COMMIT = $(shell git rev-parse HEAD || echo "unknown")
 DATE = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 MANIFEST_DIR = $(shell pwd)/deploy/manifests
```

Comment:
If we're going to have an `nginx-plus` directory, we should probably have an `nginx-oss` directory as well for the open source version.

Reply:
oh yes. I do have that, maybe you are looking at an older version of my branch.

Reply:
Hmm, I'm not seeing it. I'm only seeing the nginx-plus directory.

Reply:
This will also require an update to anywhere (docs or otherwise) that reference the manifests path to reference the new path.